### PR TITLE
add CAAPI ship order and inventory history schemas

### PR DESCRIPTION
**Pull request by jcuozzo**

### CODE REVIEW

**Review by hwalpurgis — APPROVED** [review]

File: data/docs/jekyll/api/caapi/oas_caapi.json [hunk 1]
```diff
@@ -4597,6 +4597,79 @@
           }
         }
       },
+      "Inventory History": {
```
Comment (manchuck) [inline data/docs/jekyll/api/caapi/oas_caapi.json:4600]: Having spaces can cause issues for some libs. You can set the title property if you want to display it differently

File: data/docs/jekyll/api/caapi/oas_caapi.json [hunk 2]
```diff
@@ -4632,6 +4705,120 @@
           }
         }
       },
+      "Ship Order": {
```
Comment (manchuck) [inline data/docs/jekyll/api/caapi/oas_caapi.json:4708]: Space here as well

**Review by manchuck — CHANGES_REQUESTED** [review]

**Comment by jcuozzo** [comment]
@manchuck the file you referenced is auto-generated from
```
./bin/generate.js -do docs oas --api=caapi
```
The only code changed was in data/docs/oas/v1/oas_caapi.json.  And the case matches the existing key for `"Work Request"`.  Should I change all 3 to be PascalCase there?

**Comment by manchuck** [comment]
@jcuozzo I can't remember but does updating to pascal case, cause the renderer to change to sentence case? If not then we can leave it if so then update

**Comment by jcuozzo** [comment]
@manchuck it appears the script will not update PascalCase to Sentence case

**Review by manchuck — APPROVED** [review]

**Pull request merged**